automated/linux/mmtests/mmtests.sh

```diff
@@ -21,60 +21,62 @@ declare -A env_variable_mappings=( ["dbench4"]="DBENCH" )
-  the parameter is, e.g., HEAD. If, instead, the parameter is
-  not set, then the suite present in TEST_DIR is used.
-
-  <TEST_GIT_URL>:
-  If this parameter is set, then the ${TEST_PROGRAM} suite is cloned
-  from the URL in TEST_GIT_URL. Otherwise it is cloned from the
-  standard repository for the suite. Note that cloning is done
-  only if TEST_PROG_VERSION is not empty
-
-  <TEST_DIR>:
-  If this parameter is set, then the ${TEST_PROGRAM} suite is cloned to or
-  looked for in TEST_DIR. Otherwise it is cloned to $(pwd)/${TEST_PROGRAM}
-
-  <SKIP_INSTALL>:
-  This flag controls two things: benchmark installation and benchmark's
-  dependencies installation.
-  default: false
-
-  <MMTESTS_CONFIG_FILE>:
-  MMMTests configuration file that describes how the benchmarks should be
-  configured and executed.
-
-  <MMTESTS_TYPE_NAME>:
-  MMTests test type, e.g. sysbenchcpu, iozone, sqlite, etc.
-
-  <MMTESTS_MAX_RETRIES>:
-  Maximum number of retries for the single benchmark's source file download"
+  Usage: $0 [-s] [-v <TEST_PROG_VERSION>] [-u <TEST_GIT_URL>] [-p <TEST_DIR>]
+          [-c <MMTESTS_CONFIG_FILE>] [-t <MMTESTS_TYPE_NAME>]
+          [-r <MMTESTS_MAX_RETRIES>] [-i <MMTEST_ITERATIONS>]
+
+  -v <TEST_PROG_VERSION>
+    If this parameter is set, then the ${TEST_PROGRAM} suite is cloned. In
+    particular, the version of the suite is set to the commit pointed to by the
+    parameter. A simple choice for the value of the parameter is, e.g., HEAD.
+    If, instead, the parameter is not set, then the suite present in TEST_DIR
+    is used.
+
+  -u <TEST_GIT_URL>
+    If this parameter is set, then the ${TEST_PROGRAM} suite is cloned from the
+    URL in TEST_GIT_URL. Otherwise it is cloned from the standard repository
+    for the suite. Note that cloning is done only if TEST_PROG_VERSION is not
+    empty.
+
+  -p <TEST_DIR>
+    If this parameter is set, then the ${TEST_PROGRAM} suite is cloned to or
+    looked for in TEST_DIR. Otherwise it is cloned to $(pwd)/${TEST_PROGRAM}
+
+  -s
+    This flag disables benchmark installation and benchmark's
+    dependencies installation.
+
+  -c <MMTESTS_CONFIG_FILE>
+    MMMTests configuration file that describes how the benchmarks should be
+    configured and executed.
+
+  -t <MMTESTS_TYPE_NAME>
+    MMTests test type, e.g. sysbenchcpu, iozone, sqlite, etc.
+
+  -r <MMTESTS_MAX_RETRIES>
+  Maximum number of retries for the single benchmark's source file download
+
+  -i <MMTEST_ITERATIONS>
+    The number of iterations to run the benchmark for."
 
   exit 1
 }
 
-while getopts "c:p:r:s:t:u:v:" opt; do
+while getopts "c:p:r:st:u:v:i:" opt; do
   case "${opt}" in
     c)
       MMTESTS_CONFIG_FILE="${OPTARG}"
       ;;
     p)
       if [[ "$OPTARG" != '' ]]; then
-        TEST_DIR="$OPTARG"
+        TEST_DIR="${OPTARG}"
       fi
       ;;
     r)
       MMTESTS_MAX_RETRIES="${OPTARG}"
       ;;
     s)
-      SKIP_INSTALL="${OPTARG}"
+      SKIP_INSTALL=true
```

Comment:
I would say leave it with an optarg for now.
all the other tests are using that approach too on this repository.

Reply:
This solution is better as it's more compact, no need for handler "true/True".

Reply:
Agree, +1